benchmarks/benchmark_simulariumio.py

```diff
@@ -53,20 +62,31 @@ def convert_condensate_springsalad(input_path):
 
 
 convert_benchmarks = {
-    "d_c" : convert_d_c_cytosim,
-    "springsalad_condensate" : convert_condensate_springsalad,
-}   
+    "d_c": convert_d_c_cytosim,
+    "springsalad_condensate": convert_condensate_springsalad,
+}
+
 
 def main():
     parser = argparse.ArgumentParser(
         description="Parses large data files to test speed of SimulariumIO"
     )
-    current_dir = os.getcwd()
-    for item in os.listdir(current_dir):
-        if not os.path.isdir(item):
+    # download data if there's none in resources
+    found_data = False
+    for item in os.listdir(LOCAL_RESOURCES_DIR):
+        if os.path.isdir(os.path.join(LOCAL_RESOURCES_DIR, item)):
+            found_data = True
+            break
+    if not found_data:
+        args = Args()
+        download_benchmark_resources(args)
+    # run benchmarks
+    for item in os.listdir(LOCAL_RESOURCES_DIR):
+        item_path = os.path.join(LOCAL_RESOURCES_DIR, item)
+        if not os.path.isdir(item_path):
```

Comment:
Hm, how come we need to use `item_path` instead of just `item` now?

Reply:
because the item isn't in the current working directory anymore, so we can't just use the filename, we have to include the path. Guess I could have prepended "resources/" here, but I put it all in a constant instead